### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by megies**
Trying to increase test coverage, going after some low hanging fruits..

### CODE REVIEW

**Comment by krischer** [comment]
Great effort and thoroughly needed :+1:

File: obspy/signal/tests/test_tf_misfit.py [hunk 1]
```diff
@@ -263,6 +295,30 @@ def test_plot_tfr(self):
         finally:
             np.seterr(**_t)
 
+    def test_plot_tf_misfits(self):
+        _t = np.geterr()
+        np.seterr(all="ignore")
```
Comment (QuLogic) [inline obspy/signal/tests/test_tf_misfit.py]: `with np.errstate(all='ignore'):`

I guess the other ones need fixing, too.

Comment (megies) [inline obspy/signal/tests/test_tf_misfit.py]: Changed. (It's really annoying that github is not folding comments as outdated anymore..)

**Comment by megies** [comment]
> Great effort and thoroughly needed :+1:

Euh.. already [one previously not covered bug ](https://travis-ci.org/obspy/obspy/jobs/164879214#L747) squashed.. long live coverage.. :flags:

**Comment by megies** [comment]
Seems the most recent cartopy has problems..?

**Comment by QuLogic** [comment]
Looks like an mkl problem?

**Comment by megies** [comment]
But I now forced `nomkl` through conda.. so it might be a conda packaging glitch of cartopy? Also, some fails seem mkl-unrelated, e.g. [here](https://travis-ci.org/obspy/obspy/jobs/164935726#L817)?

**Comment by megies** [comment]
Euuuh, just saw [that conda installed both `mkl` *and* `nomkl`](https://travis-ci.org/obspy/obspy/jobs/164935724#L352):
```
     mkl-11.3.3                 |                0       122.1 MB
     nomkl-1.0                  |                0          402 B
```

**Pull request merged**